packages/mds-db/migration.ts

```diff
@@ -170,6 +171,12 @@ async function alterPreviousGeographiesColumnMigration(exec: SqlExecuterFunction
   )
 }
 
+async function dropAuditEventsColumnsMigration(exec: SqlExecuterFunction) {
+  await exec(`ALTER TABLE ${schema.TABLE.audit_events} DROP COLUMN ${schema.COLUMN.provider_event_id}`)
+  await exec(`ALTER TABLE ${schema.TABLE.audit_events} DROP COLUMN ${schema.COLUMN.provider_event_type}`)
+  await exec(`ALTER TABLE ${schema.TABLE.audit_events} DROP COLUMN ${schema.COLUMN.provider_event_type_reason}`)
```

Comment:
So, these 3 columns don't exist anymore in the db, correct? They should be removed from the schema altogether then.

Reply:
Removed